protostar/commands/install/install_command.py

```diff
@@ -114,3 +117,27 @@ def install(
                 repo_dir=self._project_root_path,
                 libs_dir=libs_path,
             )
```

Comment:
should we also handle this situation?

Reply:
Ok, I think I understand this now. So if we run `install xxx`, then package `xxx` is being installed, but if we run just `install`, then the already installed packages are being updated. If I got this right, then we don't have to do anything here. Correct me if I'm wrong.